ValidationLibrary/Rules/HasNewestPtcsJenkinsLibRule.cs

```diff
@@ -19,18 +19,22 @@ public class HasNewestPtcsJenkinsLibRule : IValidationRule
         private const string JenkinsFileName = "Jenkinsfile";
 
         private const string LibraryName = "jenkins-ptcs-library";
+        private readonly string BranchName = $"feature/{LibraryName}-update";
```

Comment:
private field naming convention is to start them with an underscore. Static and const field start with an upper case letter. Fix also `PullRequestTitle`